bin/oref0-setup.sh

```diff
@@ -221,6 +221,14 @@ if openaps vendor add --path . mmeowlink.vendors.mmeowlink 2>&1 | grep "No modul
     echo Installing latest mmeowlink
     sudo pip install -U mmeowlink || die "Couldn't install mmeowlink"
 fi
+# Install Bluez for BT Tethering
```

Comment:
I think we want to make this conditional: no sense doing all the bluez stuff unless the user is trying to set it up.